src/hooks/embalmerFacet/useSubmitSarcophagus.ts

```diff
@@ -48,7 +48,7 @@ export function useSubmitSarcophagus() {
       {
         name: name,
         recipient: recipient.address,
-        resurrectionTime: resurrection / 1000,
+        resurrectionTime: resurrection / 1000, // TODO: verify contract is in seconds
```

Comment:
Huh I dunno if it was in a dream or something, but I remember Seth confirming that this is indeed seconds on the contract side, and this comment being removed? 

Interesting.

Reply:
Yes `resurrection` is in MS and the contract is in Seconds, so I removed the comment.  Do you think I should still have a comment clarifying that?

Reply:
Yeahh that'd be a good idea tbh